tests/test_database.py

```diff
@@ -172,3 +173,15 @@ def test_coordinates(db):
         print(t)
 
     assert len(t) == 0, f"{len(t)} Sources failed coordinate checks"
+
+
+def test_sig_figs_parallax(db):
+    # verify that the precision on parallax isn't greater than the error's precision
+    t = (
+        db.query(db.Parallax.c.parallax_mas, db.Parallax.c.parallax_error)
+        .astropy()
+    )
+    for i in t:
+        parallax_sig_figs = count_significant_digits_numpy(i['parallax_mas'])
+        error_sig_figs = count_significant_digits_numpy(i['parallax_error'])
+        assert error_sig_figs >= parallax_sig_figs, f"Parallax error has fewer significant figures than parallax for {i['parallax']} +/- {i['parallax_error']}"
```

Comment:
count the number of situations in which this happens (and build a table that contains them) and the assert that there are hopefully 0 occurrences.  This anticipates that we would allow this to happen, we're just drawing folks attention to it.